memote/suite/tests/test_biomass.py

```diff
@@ -57,16 +56,16 @@ def test_biomass_presence():
     future.
     """
     ann = test_biomass_presence.annotation
-    ann["data"] = BIOMASS_IDS
+    ann["data"] = [
+        rxn.id for rxn in helpers.find_biomass_reaction(read_only_model)]
     ann["message"] = wrapper.fill(
         """In this model {} the following biomass reactions were
         identified: {}""".format(
             len(ann["data"]), truncate(ann["data"])))
     assert len(ann["data"]) > 0, ann["message"]
 
 
-# TODO: Mark this test as skipped if the biomass components lack formulas!
```

Comment:
Did you open an issue for this now that you've removed it, or did you end up fixing this in one of the other PRs?

Reply:
Okay, I have added an alternative message when this is the case. I decided against skipping.